web/app/views/api.scala.html

```diff
@@ -53,6 +53,7 @@ <h2 id='query-syntax'>Abfragemöglichkeiten <small><a href='#query-syntax'><span
 	@desc("Geschachtelte Feldsuche: \"contribution.agent.label\"", resources.routes.Application.query("contribution.agent.label:Melville", format="json"))
 	@desc("In eingebetteten Objekten suchen: \"nested\"", resources.routes.Application.query("", format="json", nested="contribution:contribution.agent.label:Melville AND contribution.role.label:Autor"))
 	@desc("Bereichssuche: \"publication.startDate:[* TO 1500]\"", resources.routes.Application.query("publication.startDate:[* TO 1500]", format="json"))
+	@desc("Aggregationen über zwei Felder: \"aggregations\"", resources.routes.Application.query("kunst", aggregations="subject.id,subject.componentList.id",format="json"))
```

Comment:
I'd suggest we use just `Aggregationen: "aggregations"` as the label, since it can be any number of fields.

Reply:
done and redeployed to staging